editor.planx.uk/src/pages/FlowEditor/components/Flow/FeedbackPage.tsx

```diff
@@ -46,20 +82,168 @@ const getDetailedFeedback = async (feedbackId: number) => {
     query: GET_FEEDBACK_BY_ID_QUERY,
     variables: { feedbackId },
   });
-  console.log(detailedFeedback);
+  return detailedFeedback;
+};
+
+type FeedbackType =
+  | "issue"
+  | "idea"
+  | "comment"
+  | "inaccuracy"
+  | "helpful"
+  | "unhelpful";
+
+const feedbackTypeIcon = (type: FeedbackType) => {
+  switch (type) {
+    case "issue":
+      return { icon: <WarningIcon />, title: "Issue" };
+    case "idea":
+      return { icon: <LightbulbIcon />, title: "Idea" };
+    case "comment":
+      return { icon: <MoreHorizIcon />, title: "Comment" };
+    case "helpful":
+      return {
+        icon: <CheckCircleIcon color="success" />,
+        title: "Helpful (help text)",
+      };
+    case "unhelpful":
+      return {
+        icon: <CancelIcon color="error" />,
+        title: "Unhelpful (help text)",
+      };
+    default:
+      return { icon: <RuleIcon />, title: "inaccuracy" };
+  }
 };
 
 export const FeedbackPage: React.FC<Props> = ({ feedback }) => {
+  const displayFeedbackItems = [
+    "address",
+    "projectType",
+    "nodeTitle",
+    "nodeType",
+    "helpText",
+  ];
+
+  return (
+    <Container maxWidth="contentWrap">
+      <Box py={4}>
+        <EditorRow>
+          <Typography variant="h2" component="h3" gutterBottom>
+            Feedback log
+          </Typography>
+          <Typography variant="body1">
+            User-sumbitted feedback gathered for this service.
```

Comment:
nit: I know this is still draft, but spotted a typo "sumbitted" -> "submitted" while reviewing pizza ! 

Also, nitpick, is "Feedback from users about this service" perhaps more clear?

Reply:
Yes, clearer and simpler language